allensdk/api/queries/cell_types_api.py

```diff
@@ -80,7 +87,24 @@ def list_cells(self,
             # cell reporter status
             cell['reporter_status'] = cell['cell_reporter']['name']
 
-        return self.filter_cells(cells, require_morphology, require_reconstruction, reporter_status)
+        result = self.filter_cells(cells, require_morphology, require_reconstruction, reporter_status)
+        if id:
```

Comment:
I'd like to avoid changing the return signature of this function.  Can we move the list flattening to get_cell, since that function is explicitly returning info for just one cell?